src/libs/actions/MapboxToken.ts

```diff
@@ -38,11 +37,11 @@ const setExpirationTimer = () => {
             return;
         }
         console.debug(`[MapboxToken] Fetching a new token after waiting ${REFRESH_INTERVAL / 1000 / 60} minutes`);
-        API.read('GetMapboxAccessToken');
+        API.read('GetMapboxAccessToken', {}, {});
```

Comment:
Please test the map extensively in your PR and ensure this won't cause regressions 🙂